### ISSUE TITLE
Build 2nd Ansible 8 Alpha

### ISSUE THREAD

**Issue by samccann** [ansible-community/community-team#223]
Based on ansible-core RC1 (should be available on 4/24). Ansible 8 roadmap has this alpha available by 5/2

**Comment by gotmax23** [comment]
By now, there should have been two alphas released AIUC. ansible 8.0.0a1 to coincide with ansible-core 2.15.0b1 and ansible 8.0.0a2 to coincide with ansible-core 2.15.0b2.

**Comment by samccann** [comment]
@gotmax23 yeah we lost track of it all. That's why I'm creating issues and a [new project board ](https://github.com/orgs/ansible-community/projects/7/views/1) in the hopes of preventing this from happening again.

**Comment by gotmax23** [comment]
Ack, thanks. IMO, it would be better to transfer these issues to the ansible-build-data repository so they're more visible to the community.

**Comment by samccann** [comment]
oh hadn't thought of that, thanks!
